app/views/news_feed/index.html.erb

```diff
@@ -2,7 +2,8 @@
 
 <% @activities.each do |activity| %>
   <div class = "activity">
-  [username here]<%= link_to activity.owner.name, activity.owner if activity.owner %>
+  <%= "#{activity.trackable.user.first_name} #{activity.trackable.user.last_name}" %> 
```

Comment:
Is there a way to not have to break the law of demeter here? can we use a model method somewhere to return the full name of a user?